src/identity/masterpubkey.ts

```diff
@@ -159,6 +159,7 @@ export class MasterPublicKey extends Identity implements IdentityInterface {
         confidentialAddress: confidentialAddress!,
         blindingPrivateKey: blindingKeyPair.privateKey!.toString('hex'),
         derivationPath: path,
+        publicKey: publicKey.toString('hex'),
```

Comment:
but we have already publicKey in AddressInterfaceExtended? not needed maybe?

Reply:
Marina don't import AddressInterfaceExtended, only AddressInterface.
Start using AddressInterfaceExtended on Marina would cause a major code refactor.
But you're right, information is duplicated.